index.js

```diff
@@ -166,6 +165,12 @@ WebtaskProfile.prototype.createToken = Bluebird.method(function (options, cb) {
         params.dr = 1;
     if (options.name)
         params.jtn = options.name;
+
+    if (options.auth0)
+        params.auth = options.auth0;
+    else if (options.secure)
```

Comment:
Shouldn't this be `options.share`? Otherwise where is `options.secure` coming from?